converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -22,6 +24,10 @@
 	private String message;
 	@JsonProperty("path")
 	private String path = "";
+	@JsonProperty("line")
+	private Integer line;
+	@JsonProperty("column")
```

Comment:
What's the reason we need the `@JsonProperty` 3 times? Is this duplicate code? Maybe we need a story to fix this if it is duplicate.